pkg/acsengine/engine.go

```diff
@@ -984,6 +987,27 @@ func getDataDisks(a *api.AgentPoolProfile) string {
 	return buf.String()
 }
 
+func getEtcdDisk(orchestratorType api.OrchestratorType, m *api.MasterProfile) string {
+	if orchestratorType != api.Kubernetes {
+		return ""
+	}
+	var buf bytes.Buffer
+	buf.WriteString("\"dataDisks\": [\n")
+	etcdDisks := `            {
+              "createOption": "Empty",
+              "diskSizeGB": "%d",
+              "lun": %d,
+              "name": "[concat(variables('masterVMNamePrefix'), copyIndex(),'-etcddisk')]",
+              "vhd": {
+                "uri": "[concat(reference(concat('Microsoft.Storage/storageAccounts/',variables('masterStorageAccountName')),variables('apiVersionStorage')).primaryEndpoints.blob,'vhds/',    variables('masterVMNamePrefix'),copyIndex(),'-etcddisk.vhd')]"
+               }
+            }`
+
+	buf.WriteString(fmt.Sprintf(etcdDisks, 1, 0))
```

Comment:
Do we know the size of etcd in a 5000 node cluster or what etcd will grow to?  Perhaps we can look at Dmitry's 1200 node cluster.

Let's determine what to do in each of the standard and premium cases.  Based on pricing (https://azure.microsoft.com/en-us/pricing/details/managed-disks/):
1-128GB premium disk is $19.71 /month
1GB standard will be about $0.43 / month

I think it will be safe to go with 128GB for both.